src/odemis/acq/stream/_static.py

```diff
@@ -346,7 +346,7 @@ def _project2Polar(self, pos):
                 # TODO: could use the size of the canvas that will display
                 # the image to save some computation time.
 
-                # get bg image if existing, else "pass-through"
+                # get bg image if existing, else "pass-through" = MD_POL_NONE
```

Comment:
That comment is a little confusing. I'd reword it this way: `Get bg image, if existing. It must match the polarization (defaulting to MD_POL_NONE).`